src/web_server.cpp

```diff
@@ -229,12 +229,7 @@ void buildStatus(DynamicJsonDocument &doc) {
   doc["evse_connected"] = (int)evse.isConnected();
 
   create_rapi_json(doc);
-
-  doc["status"] = evse.getState().toString();
-
-  doc["elapsed"] = evse.getSessionElapsed();
```

Comment:
Think we will need to keep these for backward compatibiity, at least for a short period

Reply:
I thought this one was a typo I think as the properties was "session_elapsed" in evse_mon. You're sur it is intended to have twice the same value in 2 different props on 2 different place ?

Reply:
Hum... ok... LEts leave for now and group all the'depricated' ones together. Will discuss a plan for removing all of them over the next few releases but don't just want to remove with no notice.